packages/dd-trace/src/appsec/iast/taint-tracking/csi-methods.js

```diff
@@ -2,16 +2,29 @@
 
 const csiMethods = [
   { src: 'plusOperator', operator: true },
-  { src: 'trim' },
-  { src: 'trimStart', dst: 'trim' },
-  { src: 'trimEnd' },
   { src: 'concat' },
-  { src: 'substring' },
-  { src: 'substr' },
+  { src: 'replace' },
   { src: 'slice' },
-  { src: 'replace' }
+  { src: 'substr' },
+  { src: 'substring' },
+  { src: 'trim' },
+  { src: 'trimEnd' },
+  { src: 'trimStart', dst: 'trim' }
 ]
 
+function getExpectedMethods () {
```

Comment:
This function is only used in [taint-tracking-operations.spec.js](https://github.com/DataDog/dd-trace-js/pull/2805/files#diff-c814fa1c1f84c575f293e734360f543df531d957e4bac775e57cb9f967dee39eR586). I think it should live there, in the test file, not in the actual code file.